.travis.yml

```diff
@@ -7,6 +7,10 @@ services:
   - postgresql
 addons:
   postgresql: "9.5"
+  apt:
+    packages:
+      - postgresql-9.5-postgis-2.3
```

Comment:
Why do you need postgis here?

Reply:
Without it build fails with error:

```
15:08:24.533 [info]  execute "CREATE EXTENSION IF NOT EXISTS postgis"
** (Postgrex.Error) ERROR 58P01 (undefined_file): could not access file "$libdir/postgis-2.3": No such file or directory

```

Reply:
If you urgently need this, I can merge this PR. 

But in general, trailing migrations should be squashed and we need a better way to include them. Effectively having postgis in dependencies would require everybody who is using this package to enable it on PostgreSQL side.

Reply:
That would be nice, but i'm not sure i understand how the build worked before (this migration exists 5 months already https://github.com/Nebo15/ecto_trail/blob/master/priv/repo/migrations/20170621171954_add_location_to_resource_table.exs). Probably postgres image changed on travis to not include postgis extension.

Reply:
Ok, if you don't mind, i'll merge it